12-offer-encoding.md

```diff
@@ -706,23 +693,19 @@ Note that the recipient of the invoice can determine the expected
 amount from either the offer it received, or the invoice_request it
 sent, so often already has authorization for the expected amount.
 
-The default `relative_expiry` of 7200 seconds, which is generally a
+The default `invoice_relative_expiry` of 7200 seconds, which is generally a
 sufficient time for payment, even if new channels need to be opened.
 
 Blinded paths provide an equivalent to `payment_secret` and `payment_metadata` used in BOLT 11.
-Even if `node_id` is public, we force the use of blinding paths to keep these features.
+Even if `offer_node_id` is public, we force the use of blinding paths to keep these features.
 If the recipient does not care about the added privacy offered by blinded paths, they can create a path of length 1 with only themselves.
 
 Rather than provide detailed per-hop-payinfo for each hop in a blinded path, we aggregate the fees and CLTV deltas.
 This avoids trivially revealing any distinguishing non-uniformity which may distinguish the path.
 
-It's often useful to provide capacity hints, particularly where more
-than one blinded path is included, for payers to use multi-part
-payments.
+The invoice issuer is allowed to ignore `invoice_request_payer_note` (it has an odd
+number, so is optional), but must still copy it exactly.
```

Comment:
I'm a little confused by the use of an odd number for `invoice_request_payer_note`. I thought odd meant it is ok if you don't know how to interpret this type (i.e., the implementation predates this type). Not this is optional.

Reply:
Yes, language here was too loose.  It's "optional to understand"...